mediapicker/src/main/java/org/wordpress/android/mediapicker/viewmodel/MediaPickerViewModel.kt

```diff
@@ -577,18 +597,46 @@ internal class MediaPickerViewModel @Inject constructor(
         }
     }
 
+    fun checkMediaPermissions(permissions: List<PermissionsRequested>, isAlwaysDenied: Boolean) {
+        if (!mediaPickerSetup.isImagesPermissionRequired &&
+            !mediaPickerSetup.isVideoPermissionRequired &&
+            !mediaPickerSetup.isAudioPermissionRequired
+        ) {
```

Comment:
Yes, that makes sense. This was the result of copying and refactoring the original implementation.

Reply:
> Is having the ability to request specific media permission not necessary for the project?

You can do that, you just have to create the `MediaPickerSetup` instance yourself instead of using the `DeviceMediaPickerSetup`, for example. See my other [comment](https://github.com/wordpress-mobile/WordPress-MediaPicker-Android/pull/63#discussion_r1153450761).